Protectli VP24xx overview content (#615)
Dasharo/docs: docs/variants/protectli_vp2420/overview.md

# Overview

The Vault is a small form network appliance built for use as a firewall

>  / router, virtualization platform, a daily-driven personal computer,

Reviewer: ```suggestion
/ router, virtualization platform, a daily-driven personal computer,
```

>  and more. The VP2420 is another addition to the Vault Pro series. It
>  is a 4 x 2.5 G network port design that leverages a low power,
>  but versatile Intel Celeron J6412 CPU.

Reviewer: ```suggestion
and more. The VP2420 is based on a 4 x 2.5 G network port design that
leverages a low power, but versatile Intel Celeron J6412 CPU.
```

> ![](/images/VP2410.png)

Reviewer: It should be VP2420 image right?
